lesson-1/farzad/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
```

Comment:
It's better if you use HTML5 meta tags 
you don't need to remember them you should know the reason why we need them and use vscode `html snippets` 
```suggestion
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Logo</title>
        <link rel="stylesheet" href="./styles.css" />
</head>
```